README.md

```diff
@@ -36,6 +36,9 @@ Variables without a default value and not marked optional, *MUST* be defined for
 | GP2GP_AMQP_MAX_REDELIVERIES          | 3                         | The number of times an message will be retried to be delivered to consumer. After exhausting all retires, it will be put on DLQ.<queue_name> dead letter queue
 | GP2GP_MHS_INBOUND_QUEUE              | inbound                   | Name of the queue for MHS inbound
 | GP2GP_TASK_QUEUE                     | gp2gpTaskQueue            | Defines name of internal taskQueue.
+| GP2GP_GPC_GET_URL                    | 8080                      | The URL used for GP Connect requests.
+| GP2GP_GPC_GET_STRUCTURED_ENDPOINT    | 8080                      | The endpoiint for GP Connect Get Structured Access. 
+| GP2GP_GPC_HOST                       | 8080                      | The host used for GP Connect requests.
```

Comment:
The Default for these is not 8080. I'm fine with leaving them blank in the README.